### PR TITLE
Integrate GolangCI-Lint

### PR DESCRIPTION
**Pull request by arshchimni**
#### Summary
Includes all the changes that were required to integrate goLang-Ci lint.

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-webex/issues/45

### CODE REVIEW

File: server/webex/client.go [hunk 1]
```diff
@@ -154,20 +153,19 @@ func (c *client) roundTrip(payload string) (*bytes.Buffer, error) {
...
-	if r.Body != nil {
-		ioutil.ReadAll(r.Body)
-		r.Body.Close()
-	}
-}
+// func closeBody(r *http.Response) {
+// 	if r.Body != nil {
+// 		r.Body.Close()
+// 	}
+// }
```
Comment (hanzei) [inline server/webex/client.go:160]: Can this method get deleted?

**Review by hanzei — APPROVED** [review]
LGTM, one question below

File: server/webex/client.go (hunk 1 above)
Comment (arshchimni) [reply server/webex/client.go:160]: Sure I will delete it

**Review by hanzei — APPROVED** [review]

**Comment by hanzei** [comment]
@arshchimni Heads up that @cpoile is out is week. He will review your PR once he is back.

**Comment by arshchimni** [comment]
> @arshchimni Heads up that @cpoile is out is week. He will review your PR once he is back.

Thanks for the head up 😄

**Comment by hanzei** [comment]
@cpoile Could you please take a look at this PR?

**Review by cpoile — APPROVED** [review]
Sorry for the delay on this. Nice work, thank you @arshchimni !

**Comment by arshchimni** [comment]
> Sorry for the delay on this. Nice work, thank you @arshchimni !

Thank you! 😄

**Comment by hanzei** [comment]
@DHaussermann Not sure if there is much to test here. There are no functional changes.

**Review by DHaussermann — APPROVED** [review]
Tested and passed
- Ran full `make` 
- No linting issue or other build issues found
- Deployed and enabled the plugin as a precaution
No issues found
LGTM!

Thanks @arshchimni  for this enhancement. Much appreciated!

**Pull request merged**